app/javascript/components/editor/TestRunSummary.tsx

```diff
@@ -1,25 +1,23 @@
 import React from 'react'
-import { TestRun, TestRunnerStatus, TestRunStatus, TestStatus } from './types'
+import { TestRun, TestRunner, TestRunStatus, TestStatus } from './types'
 import { TestRunSummaryByStatusHeaderMessage } from './TestRunSummaryByStatusHeaderMessage'
 import { TestRunOutput } from './TestRunOutput'
 import { SubmitButton } from './SubmitButton'
 import { GraphicalIcon } from '../common'
 
 export const TestRunSummary = ({
   testRun,
-  testRunnerStatus,
+  testRunner,
   onSubmit,
   isSubmitDisabled,
   onCancel,
-  averageTestDuration,
   showSuccessBox,
 }: {
   testRun: TestRun
-  testRunnerStatus?: TestRunnerStatus
+  testRunner: TestRunner
   onSubmit?: () => void
   isSubmitDisabled?: boolean
   onCancel?: () => void
-  averageTestDuration?: number
```

Comment:
It said here that the average test duration could be null or undefined, whereas in fact it could not be. For tracks that don't have it set explicitly in their `config.json`, a default value is used.